gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/model/GaaSCreateRequestBody.java

```diff
@@ -16,37 +16,58 @@
 
 package uk.gov.gchq.gaffer.gaas.model;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Pattern;
-import java.io.Serializable;
+import java.util.List;
 import java.util.Map;
 
 /**
  * <b>GaaS: Create Gaffer Request Body</b>
  */
-public class GaaSCreateRequestBody implements Serializable {
+public class GaaSCreateRequestBody {
 
+    @JsonProperty("graphId")
     @NotNull(message = "Graph id should not be null")
     @NotBlank(message = "Graph id should not be null")
     @Pattern(regexp = "[a-z0-9]*$", message = "Graph ID can contain only digits or lowercase letters")
```

Comment:
Can this be changed to allow either lower case characters or dashes so multiple words can be represented?

Reply:
Have added upper case characters so multiple words can be represented using camel case

Reply:
Actually have realised that Kubernetes wants only lowercase letters or digits and no dashes that's why the pattern was set for only digits and lower case letters so have reverted back.